source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/projects/show/reviewers/index.tsx

```diff
@@ -31,8 +31,9 @@ class Reviewers extends React.Component<IProps> {
     isAddFormVisible: false
   };
 
-  toggleAddForm = () => {
-   this.setState({ isAddFormVisible: !this.state.isAddFormVisible });
+  toggleAddForm = (e) => {
+    e.preventDefault();
+    this.setState({ isAddFormVisible: !this.state.isAddFormVisible });
```

Comment:
the toggle helper from react-state-helpers: https://www.npmjs.com/package/react-state-helpers

Reply:
yeah eventually :D